### ISSUE TITLE
[Bug] "Home" doesn't display videos or titles when launched and contents are jumpy when scrolling

### ISSUE THREAD

**Issue by HaruMatsu9124** [arichornlover/uYouEnhanced#565]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### iOS/iPadOS version

iOS 17.5 Beta 3

### Device model

iPhone 14 Pro Max

### Sideload tool

Sidestore

### Tweak version

v19.14.2-3.0.3

### Describe the bug

 "Home" doesn't display videos or titles when launched and contents are jumpy when scrolling
This is with default settings. Stuff that messes with Home feed is not enabled (like hide shorts or community posts).
![ACE07508-CB4A-467B-9F9A-B7A957E362E0](https://github.com/arichornlover/uYouEnhanced/assets/79664720/acb6c3a4-7ac6-43de-bc48-9c5c7ae3c3c6)

https://github.com/arichornlover/uYouEnhanced/assets/79664720/05e97ff5-1045-4608-8e60-cea9e314d0f2

### Steps to reproduce the issue

1. There are no steps. As soon as I launch the app it occures.
2. 
3. 

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by Dburkhart61** [comment]
Same thing for me and many others it was working fine than all of a sudden its broke

**Comment by jeayalar** [comment]
Yup, I’m also experiencing the same issue. Happened less than 24 hours ago for me. Weird.

**Comment by HaruMatsu9124** [comment]
There seems to be no fix at the moment. Tried reinstalling, double checking no options that messes with Home feed is enabled, like “Hide Short Cells” or “Hide Community Posts” and even tried “Disable Modern A/B flags” but to no avail. My theory is that it’s a change rolled out by YouTube on server side. Possibly A/B testing of some sort. Could also have something to do with Google’s “Crackdown on “Third-Party clients that disables ads” but I have no idea.

**Comment by jianmingyong** [comment]
https://github.com/arichornlover/uYouEnhanced/issues/529#issuecomment-2065625732

**Comment by Dburkhart61** [comment]
> https://github.com/arichornlover/uYouEnhanced/issues/529#issuecomment-2065625732

This doesnt fox for me im using the tweak and i downgraded to that version and app crashes

**Comment by Kaiok3nn** [comment]
> > [#529 (comment)](https://github.com/arichornlover/uYouEnhanced/issues/529#issuecomment-2065625732)
> 
> This doesnt fox for me im using the tweak and i downgraded to that version and app crashes

Are you sure you marked the "enable app version spoofer" toggle too? I missed that and thought it didn't work either lol

**Comment by Dburkhart61** [comment]
Worked like a charm thanks!

**Comment by emlodro** [comment]
same issue here. the spoof workaround works on my iphone with signulous but not my ipad (via trollstore) for some reason

**Comment by PupperooX** [comment]
Same issue

**Comment by JtMotoX** [comment]
I had this issue and was able to fix it by disabling the "Remove YouTube Ads" setting. My guess is that it thinks all the videos are ads and hides them.

Settings -> General -> uYou -> Other Settings -> Remove YouTube Ads
